Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -115,6 +150,11 @@ const ContestsProvider = ({ children }: IContestsProviderProps) => {
         [ clearParams ],
     );
 
+    const clearSorting = useCallback(
+        () => clearParams(),
```

Comment:
Is this correct? Won't this clear the filters as well?

Reply:
The same for `clearFilters()`. Won't it clear the sorting?

Reply:
I removed clearSorting and left clearFilters only, since they do the same.